tests/unit_tests/parsers/event/test_append_event.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import pytest
+
+from gcf_data_mapper.enums.event import Event
+from gcf_data_mapper.parsers.event import append_event
+
+
+@pytest.fixture
+def mock_event():
+    return Event(1, "Approved", "ApprovalDate")
+
+
+@pytest.fixture
+def mock_row():
+    return pd.Series({"ApprovalDate": "2023-01-01"})
```

Comment:
does this fixture override the one the one that you have defined in the `conftest.py` of the same name ? @katybaulch

Reply:
Overrides :)